Source/DataDictionary/gdcmPrivateDefaultDicts.cxx

```diff
@@ -1901,83 +1901,83 @@ static const DICT_ENTRY DICOMV3DataDict [] = {
-  {0x200d,0x0011,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x0012,"Eclispe 60",VR::OB,VM::VM1,"Private data",false },
-  {0x200d,0x0013,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x0014,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x0015,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x001a,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x001b,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x0020,"Eclispe 60",VR::SQ,VM::VM1,"Private data",false },
-  {0x200d,0x0027,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0028,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0030,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0031,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0032,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0033,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0034,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0035,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0036,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0037,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0038,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0039,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x003a,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x200d,0x0050,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x0051,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x0052,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0053,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x0054,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x0055,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0056,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0057,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x0058,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x200d,0x0059,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x005a,"Eclispe 60",VR::FD,VM::VM1,"Private data",false },
-  {0x200d,0x005d,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x005e,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x200d,0x00ff,"Eclispe 60",VR::IS,VM::VM1,"Private data",false },
-  {0x7777,0x0001,"Eclispe 60",VR::DA,VM::VM1,"Private data",false },
-  {0x7777,0x0002,"Eclispe 60",VR::CS,VM::VM1,"Private data",false },
-  {0x7777,0x0003,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x7777,0x0010,"Eclispe 60",VR::LO,VM::VM1,"Private data",false },
-  {0x7777,0x0014,"Eclispe 60",VR::LT,VM::VM1,"Private data",false },
+  {0x0029,0x0030,"Eclipse 60",VR::UL,VM::VM1,"Private data",false },
+  {0x0029,0x0031,"Eclipse 60",VR::UL,VM::VM1,"Private data",false },
```

Comment:
don't do that please

Reply:
nevermind, it turns out codespeall caught an actual bug

Reply:
Yes, I thought this could be the creative name of an imaging device, but I looked it up, and it turns out that some Philips MRI scanners used to be actually named  **Eclipse**, not **Eclispe**. But then I am not familiar with these scanners, I don't know whether _Eclipse 60_ is an actual Philisp model.